src/components/core/pivotTable/ResizedColumnsMap.ts

```diff
@@ -0,0 +1,183 @@
+// (C) 2007-2020 GoodData Corporation
+// tslint:disable:prefer-conditional-expression
+import { Execution } from "@gooddata/typings";
+import { ColDef, Column } from "ag-grid-community";
+import omit = require("lodash/omit");
+
+import { getColumnIdentifier, isMeasureColumn } from "./agGridUtils";
+import { getColumnWidthsFromMap, convertColumnWidthsToMap } from "./agGridColumnSizing";
+import {
+    ColumnEventSourceType,
+    ColumnWidthItem,
+    isAllMeasureColumnWidthItem,
+    IAllMeasureColumnWidthItem,
+    ColumnWidth,
+    IResizedColumnsItem,
+    isColumnWidthValue,
+    isColumnWidthAuto,
+} from "../../../interfaces/PivotTable";
+
+export interface IResizedColumnsCollection {
+    [columnIdentifier: string]: IResizedColumnsCollectionItem;
+}
+
+export interface IResizedColumnsCollectionItem {
+    width: ColumnWidth;
+    source: ColumnEventSourceType;
+}
+
+export class ResizedColumnsMap {
```

Comment:
Maybe this should be called somehow different? Maybe Store, Repository, or Service, rather than Map? 

or maybe even split into different classes, it seems it's doing a lot of different things

Reply:
skipped

Reply:
not even `ResizedColumnsStore`? I think the `Map` is really confusing